### PR TITLE
[STRATCONN] 3183 Added email subscribe as dynamic field in braze

### PR DESCRIPTION
**Pull request by AnkitSegment**
_A summary of your pull request, including the what change you're making and why._

Added dynamic field (Email Subscribed) for braze cloud action mode destination and braze browser mode destinations
JIRA TICKET: https://segment.atlassian.net/browse/STRATCONN-3183?atlOrigin=eyJpIjoiOTMzMzllNWJhMTJlNGU2NDk5MDkzNmQ5OGUwMDM0ZWIiLCJwIjoiaiJ9

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

Screenshot 2023-09-25 at 4 36 31 AM

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/browser-destinations/destinations/cdpresolution/src/generated-types.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+   * Client identifier provided by CDP Resolution [Hashed Account ID]
    */
-  endpoint: string
+  clientIdentifier: string
   /**
-   * Client identifier provided by CDP Resolution [CDP Resultion Account](https://www.cdpresolution.com/accountsettings)
+   * Identity resolution endpoint. [CDP Resolution documentation](https://www.cdpresolution.com/docs/)
    */
   clientIdentifier: string
-}
```
Comment (varadarajan-tw) [inline packages/browser-destinations/destinations/cdpresolution/src/generated-types.ts:12]: Could you remove these unrelated changes? I would suggest you a create a separate PR for including these changes.

**Review by varadarajan-tw — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @AnkitSegment , I just noticed that you merged this PR. Do you want it deployed with the next Cloud and Web deploys?
If so, can you label it with "Ready for Release" please?

**Comment by joe-ayoub-segment** [comment]
Note: agreed with @AnkitSegment over Slack that this PR is ready for release.

**Comment by joe-ayoub-segment** [comment]
Hi @AnkitSegment - I believe that this was deployed yesterday by @maryamsharif . Waiting to confirm if Maryam ran a Push.

**Comment by joe-ayoub-segment** [comment]
Hi @AnkitSegment , the rollout should be complete later today.
